dev/tests/behavioral/features/front/product-purchase.feature

```diff
@@ -0,0 +1,62 @@
+@javascript
+Feature: Product purchase
+
+  Scenario: Purchase products
+
+    Given I am on "/index.php/women.html"
+
+    And I wait for element with xpath "//*[@id='maincontent']/div[4]/div[2]/div/div/ul[1]/li[1]/a" to appear
+    And I click on the element with xpath "//*[@id='maincontent']/div[4]/div[2]/div/div/ul[1]/li[1]/a"
+
+    #Choosing 1 item
+    And I wait for element with xpath "//*[@id='maincontent']/div[3]/div[1]/div[3]/ol/li[3]/div/a/span/span/img" to appear
+    And I click on the element with xpath "//*[@id='maincontent']/div[3]/div[1]/div[3]/ol/li[3]/div/a/span/span/img"
+    And I wait for page to load "/index.php/autumn-pullie.html"
+
+    And I wait for element with xpath "//*[@id='product-options-wrapper']/div/div/div[1]/div/div[3]" to appear
+    And I click on the element with xpath "//*[@id='product-options-wrapper']/div/div/div[1]/div/div[3]"
+
+    And I wait for element with xpath "//*[@id='product-options-wrapper']/div/div/div[2]/div/div[1]" to appear
+    And I click on the element with xpath "//*[@id='product-options-wrapper']/div/div/div[2]/div/div[1]"
+
+    #Add to card
+    And I wait for element with xpath "//*[@id='product-addtocart-button']/span" to appear
+    And I click on the element with xpath "//*[@id='product-addtocart-button']/span"
+
+    #Check the card
+    And I wait for element with xpath "//div[1]/header/div[2]/div[1]/a/span[2]" to appear
+    And I click on the element with xpath "//div[1]/header/div[2]/div[1]/a/span[2]"
+
+    And I wait for element with xpath "//*[@id='top-cart-btn-checkout']" to appear
+    And I click on the element with xpath "//*[@id='top-cart-btn-checkout']"
+
+    #Checkout
+    Then I wait for page to load "/index.php/checkout/"
+    And I wait for element with xpath "//*[@id='customer-email']" to appear
+
+    And I fill in the following:
+      | customer-email     | test@test.com   |
+      | firstname          | Peter           |
+      | lastname           | Smith           |
+      | company            | AAA             |
+      | street[0]          | 5 Main street   |
+      | city               | King            |
+
+    And I select "Florida" from "region_id"
+    And I fill in the following:
+      | postcode           | 12345           |
+    And I select "United States" from "country_id"
+    And I fill in the following:
+      | telephone          | 4166666666      |
+    And I click on the element with xpath "//*[@id='checkout-shipping-method-load']/table/tbody/tr[1]/td[1]/input"
```

Comment:
Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/